app/src/main/java/com/group80/uoftinder/firebase/realtime/UserRealtimeDbFacade.java

```diff
@@ -0,0 +1,77 @@
+package com.group80.uoftinder.firebase.realtime;
+
+import com.group80.uoftinder.entities.User;
+
+import java.util.List;
+
+/**
+ * A Facade for methods regrading User in realtime database. All methods are constants, and no
+ * instances of this Facade should be created.
+ */
+public class UserRealtimeDbFacade {
```

Comment:
What was the reasoning behind making all methods in this class static? Did you consider the alternative, instantiating a ```UserRealtimeDbFacade``` object to call methods on?